dolphinscheduler-ui/src/views/login/index.tsx

```diff
@@ -170,6 +178,15 @@ const login = defineComponent({
               </NButton>
             </a>
           </div>
+            {this.oauth2Providers.length > 0 ? <NDivider >
+              {this.t('login.otherwayLogin')}
+            </NDivider> : <div></div>}
```

Comment:
Don't use Ternary operator, Please user this.oauth2Providers.length > 0 && <NDivider>
              {this.t('login.otherwayLogin')}
            </NDivider>